src/components/tetris/Tetris.module.scss

```diff
@@ -86,6 +86,52 @@
   }
 }
 
+@media (any-hover: none) {
```

Comment:
This has been removed in the original main branch. 
Did you pull the updated main before merging?
Can safely be removed